doc/adr006-modules.lit.md

```diff
@@ -556,24 +559,36 @@ Using this syntax, our example looks like follows:
 
 ```bluespec
 module fixed {
-  include general(N = 3) as I3
-  include general(N = 5) as I5
-  
+  import general(N = 3).*
+  import general(N = 5) as I5
   ...
 }
 ```
 
 We believe that this syntax better reflects the nature of instances in Quint:
 
+ - If a module does not have parameters, then there is no obvious need for
+   creating a new instance. In this case, the syntax goes back to the standard
+   import.
+
+ - In the case one wants to create two identical copies of the same state machine,
+   they can introduce a fake parameter such as the name of an instance.
+
  - All names of an instance are included into the namespace of the containing
-   module, possibly with a prefix, e.g., with `I3` or `I5`.
+   module, possibly with a prefix, e.g., `I5`.
 
  - Some names replaced with expressions, e.g., `N`.
 
- - The behavior is different from `import`.
-
-Interestingly, it is hard to find a direct analogy of `include` in the modern
+Interestingly, it is hard to find a direct analogy of this construct in the modern
 programming languages. In TLA<sup>+</sup>, `INSTANCE` behaves as a safer
 version of `C++` namespaces combined with `#include`, providing semantic tests
-for action operators. Although the word `include` is not as nice as `instance`,
-the word `include` reflects the behavior of this construct better.
+for action operators.
+
+One potential problem with the approach of using import-like behavior is that
+the names that are introduced via this construct are not automatically
+exported. We can imagine that sometimes automatic export would be useful.
+Actually, the same occurs in programming languages, e.g., in TypeScript.  This
+is why TypeScript has the [export import][] combo. If we see a need for such
+behavior, we may add it in the future.
```

Comment:
I'm currently trying to write tendermint tests with instances and it's quite hard without the exporting. But I think the `import general(N = 5).*` syntax will fill the gap. Then we can do:

```bluespec
module TendermintAcc { ... }

module TendermintTest {
  import TendermintAcc.*

  ...
}

module Tendermint_n4_f1_Test {
  import TendermintAcc(
    Corr = Set("p1", "p2", "p3"),
    Faulty = Set("p4"),
    N = 4,
    T = 1,
    ValidValues = Set("v0", "v1"),
    InvalidValues = Set("v2"),
    MaxRound = 4
  ).*

  import TendermintTest.*
}
```

This would require a CLI `test` call for each instance since the CLI only tests a single module at a time. There should be other alternatives, this is one that doesn't require typing namespaces.

Reply:
That's interesting. But I am not sure it works as expected. Would not it make sense to instantiate `TendermintTest` instead of `TendermintAcc`?

Reply:
Sorry, that what I meant.